test/resque_test.rb

```diff
@@ -12,7 +12,8 @@ class ResqueTest < MiniTest::Spec
   it "enqueues normal jobs" do
     Resque.enqueue FakeJob, "x"
     Resque.enqueue FakeJob, "x"
-    assert_equal 2, Resque.size(:normal)
+    queue_name = FakeJob.instance_variable_get(:@queue)
```

Comment:
In all these tests, I prefer to simply hard-code the queue name rather than accessing an internal instance variable. So don't change the tests to use `instance_variable_get`. There's no reason to change any of these tests except line 47.